src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -199,139 +199,154 @@ export const BaseComment = ({
     return (
         // TODO: Change className to bcs-Comment once FF is removed
         <div className="bcs-BaseComment">
-            <Media
+            <div
                 className={classNames('bcs-Comment-media', {
                     'bcs-is-pending': isPending || error,
                 })}
             >
-                <Media.Figure>
-                    <Avatar
-                        getAvatarUrl={getAvatarUrl}
-                        user={createdByUser}
-                        badgeIcon={
-                            annotationActivityLink && (
-                                <IconAnnotation title={<FormattedMessage {...messages.annotationBadge} />} />
-                            )
-                        }
-                    />
-                </Media.Figure>
-                <Media.Body>
-                    {isMenuVisible && (
-                        <TetherComponent
-                            attachment="top right"
-                            className="bcs-Comment-deleteConfirmationModal"
-                            constraints={[{ to: 'scrollParent', attachment: 'together' }]}
-                            targetAttachment="bottom right"
-                        >
-                            <Media.Menu
-                                isDisabled={isConfirmingDelete}
-                                data-testid="comment-actions-menu"
-                                dropdownProps={{
-                                    onMenuOpen: () => onSelect(true),
-                                    onMenuClose: handleMenuClose,
-                                }}
-                                menuProps={{
-                                    'data-resin-component': ACTIVITY_TARGETS.COMMENT_OPTIONS,
-                                }}
+                <div className="bcs-BaseComment-MainComment">
+                    <div className="bcs-BaseComment-MetaRow">
```

Comment:
This would be a good candidate for extraction. As long as we can make sure we stay within our timeframes I'm good with it.